### ISSUE TITLE
PIA: Error opening configuration file: /etc/openvpn/pia/netherlands.ovpn

### ISSUE THREAD

**Issue by avillev** [haugene/docker-transmission-openvpn#1867]
## Before creating this issue I have:

**REQUIRED**
- [X] Read through the pinned issues for related problems
- [X] Searched for similar [provider issues][pi] and [container issues][ci]
- [X] [Read the documentation][rd], especially the troubleshooting section and FAQ
- [X] Tried to add as much relevant information to the issue as possible
- [X] Verified I have tried using newest release as well
- [X] Agreed that my issue will be closed if I do not follow this template and will remain closed until I complete the template

[pi]: https://github.com/haugene/vpn-configs-contrib/issues
[ci]: https://github.com/haugene/docker-transmission-openvpn/issues
[rd]: https://haugene.github.io/docker-transmission-openvpn/

### Container version & last working release
problem occurs in  ```3.6```

*If possible, last working version:* 
```3.6``` some time ago

### Describe the problem

```txt
The container will fail opening the opvpn file and restart in a loop. It used to work, I did not really change much. Maybe the files are not downloadable or present anymore?
```

### Add your docker run command or docker-compose file or env details

```txt
docker run -d --restart=always \
--cap-add=NET_ADMIN \
--device=/dev/net/tun \
--name "transmission" \
-p 9091:9091 \
--mount type=bind,src=/volume1/docker,dst=/docker \
--mount type=bind,src=/volume1/Downloads/transmission,dst=/data \
-e PUID="" \
-e PGID="" \
-e OPENVPN_USERNAME="xxxx" \
-e OPENVPN_PASSWORD="xxxx" \
-e OPENVPN_PROVIDER="PIA" \
-e PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin" \
-e GLOBAL_APPLY_PERMISSIONS="true" \
-e TRANSMISSION_HOME="/docker/transmission-home" \
-e TRANSMISSION_RPC_PORT="9091" \
-e TRANSMISSION_DOWNLOAD_DIR="/data/downloads/completed" \
-e TRANSMISSION_INCOMPLETE_DIR="/data/downloads/incomplete" \
-e TRANSMISSION_WATCH_DIR="/docker/transmission-home/watch" \
-e CREATE_TUN_DEVICE="true" \
-e DROP_DEFAULT_ROUTE="" \
-e WEBPROXY_ENABLED="false" \
-e WEBPROXY_PORT="8888" \
-e WEBPROXY_USERNAME="" \
-e WEBPROXY_PASSWORD="" \
-e LOG_TO_STDOUT="false" \
-e HEALTH_CHECK_HOST="google.com" \
-e REVISION="c9200cba1d5599622395f0819a9d0b862eaf3741" \
-e OPENVPN_CONFIG="netherlands" \
-e TRANSMISSION_RPC_USERNAME="xxxx" \
-e TRANSMISSION_RPC_PASSWORD="xxxx" \
-e TRANSMISSION_RPC_AUTHENTICATION_REQUIRED="true" \
haugene/transmission-openvpn:3.6
```

### Logs

```txt
Starting container with revision: acf1261567b01b3e31bf4f7e8bc119284aaa0240
Creating TUN device /dev/net/tun
mknod: /dev/net/tun: File exists
Using OpenVPN provider: PIA
Provider PIA has a custom setup script, executing it
Downloading OpenVPN config bundle openvpn into temporary file /tmp/tmp.OAMIOh
Extract OpenVPN config bundle into PIA directory /etc/openvpn/pia
Starting OpenVPN using config netherlands.ovpn
Modifying /etc/openvpn/pia/netherlands.ovpn for best behaviour in this container
Setting OpenVPN credentials...
Options error: In [CMD-LINE]:1: Error opening configuration file: /etc/openvpn/pia/netherlands.ovpn
```

### Host system

```txt
Synology DSM 7.0
Docker 20.10.3-1233
```

**Comment by avillev** [comment]
I confirm that it is duplicated from the one above but more importantly #1839 

Unchecking "execute container using high privilege" and re-checking it again fixes the issue with DSM 7.0 after migration.

**Comment by martin4eel** [comment]
Hi guys, 
I am having a similar issue... Unchecking and re-checking high priv does not work... 

DSM 7.0-41890
Transmission: Haugene Transmission (latest version, updated 9 days ago) 
https://registry.hub.docker.com/r/haugene/transmission-openvpn/

However, it seems to be something related to access:
"Error loading shared library liblzo2.so.2: Permission denied (needed by /usr/sbin/openvpn)"

Config: 
--cap-add=NET_ADMIN \
--device=/dev/net/tun \
-p 9091:9091 \
--mount type=bind,src=/volume1/docker,dst=/docker \
--mount type=bind,src=/volume1/Downloads/transmission,dst=/data \
-e PUID="1026" \
-e PGID="101" \
-e OPENVPN_USERNAME="xxxx" \
-e OPENVPN_PASSWORD="xxxx" \
-e OPENVPN_PROVIDER="NORDVPN" \
-e NORDVPN_COUNTRY="SE"
-e TZ=Europe/Stockholm
-e PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin" \
-e GLOBAL_APPLY_PERMISSIONS="true" \
-e TRANSMISSION_HOME="/docker/transmission-home" \
-e TRANSMISSION_RPC_PORT="9091" \
-e TRANSMISSION_DOWNLOAD_DIR="/data/downloads/completed" \
-e TRANSMISSION_INCOMPLETE_DIR="/data/downloads/incomplete" \
-e TRANSMISSION_WATCH_DIR="/docker/transmission-home/watch" \
-e CREATE_TUN_DEVICE="true" \
-e DROP_DEFAULT_ROUTE="" \
-e WEBPROXY_ENABLED="false" \
-e WEBPROXY_PORT="8888" \
-e WEBPROXY_USERNAME="" \
-e WEBPROXY_PASSWORD="" \
-e LOG_TO_STDOUT="false" \
-e HEALTH_CHECK_HOST="google.com" \
-e REVISION="c9200cba1d5599622395f0819a9d0b862eaf3741" \
-e TRANSMISSION_RPC_USERNAME="xxxx" \
-e TRANSMISSION_RPC_PASSWORD="xxxx" \
-e TRANSMISSION_RPC_AUTHENTICATION_REQUIRED="true" \
-e OPENVPN_OPTS="--inactive 3600 --ping 10 --ping-exit 60"



LOG OUTPUT: 

Starting container with revision: 73ec516cc246972289c7b96ffa88c81e037fe364
Creating TUN device /dev/net/tun
mknod: /dev/net/tun: File exists
Using OpenVPN provider: NORDVPN
Provider NORDVPN has a custom setup script, executing it
2021-07-09 12:48:45 Checking curl installation
2021-07-09 12:48:45 Removing existing configs
2021-07-09 12:48:45 Selecting the best server...
2021-07-09 12:48:45 Searching for group: legacy_p2p
2021-07-09 12:48:45 Searching for technology: openvpn_udp
2021-07-09 12:48:45 Best server : se513.nordvpn.com
2021-07-09 12:48:45 Downloading config: default.ovpn
2021-07-09 12:48:45 Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_udp/servers/se513.nordvpn.com.udp.ovpn
2021-07-09 12:48:45 Selecting the best server...
2021-07-09 12:48:45 Searching for country : SE (208)
2021-07-09 12:48:45 Searching for group: legacy_p2p
2021-07-09 12:48:45 Searching for technology: openvpn_udp
2021-07-09 12:48:45 Best server : se513.nordvpn.com
2021-07-09 12:48:45 Downloading config: se513.nordvpn.com.ovpn
2021-07-09 12:48:45 Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_udp/servers/se513.nordvpn.com.udp.ovpn
Starting OpenVPN using config se513.nordvpn.com.ovpn
Modifying /etc/openvpn/nordvpn/se513.nordvpn.com.ovpn for best behaviour in this container
Setting OpenVPN credentials...
adding route to local network 192.168.1.0/24 via 172.17.0.1 dev eth0
Error loading shared library liblzo2.so.2: Permission denied (needed by /usr/sbin/openvpn)
Error loading shared library libssl.so.1.1: Permission denied (needed by /usr/sbin/openvpn)
Error loading shared library libcrypto.so.1.1: Permission denied (needed by /usr/sbin/openvpn)
Error relocating /usr/sbin/openvpn: TLS_method: symbol not found

Then it spams "Symbol not found" on every row..
